### ISSUE TITLE
Handle file subselect from very large storage deals

### ISSUE THREAD

**Issue by andrewxhill** [textileio/powergate#400]
- User adds a very large directory containing many files
- Later, they want to allow subselect (coming feature in fil?) functionality so they can request per file to add to Hot layer. 

Not certain how the configs could expand to handle this, so adding this ticket to track thinking around subselect capabilities between hot/cold layer. 

```
the Hot Storage now is fetching things from the IPFS node. We can now fetch partial parts of the DAG, since al the dag is pinned.
For selects, that's more about go-ipfs supporting ipld-selectors I guess.
So, in short, sounds possible and some things might depend on the IPFS node api supporting that kind of stuff.
```
